scylla-cql/src/frame/request/batch.rs

```diff
@@ -14,6 +14,7 @@ use crate::frame::{
 const FLAG_WITH_SERIAL_CONSISTENCY: u8 = 0x10;
 const FLAG_WITH_DEFAULT_TIMESTAMP: u8 = 0x20;
 
+#[derive(Debug)]
```

Comment:
I've seen many times in discussions over Rust codebases that `Debug` should be always available whenever possible (and we are not in a strongly limited environment such as embedded). For instance: https://github.com/rust-lang/rust/pull/32054#issuecomment-192564644

Reply:
Well, the hit in compile time is not ideal. Perhaps it makes sense to enable it for public types that users interact with directly, though I'm not sure if it we should do it for non-public ones (I treat the types from scylla-cql to be kind of non-public).

Regardless, if the addition of Debug derives is not strictly necessary for the PR then it should be sent separately.

Reply:
Regarding visibility of `scylla-cql` entities, I've always had an approach that this is a low-level crate that could be used by developers on its own for their own purposes, and implementing `Debug` may come in handy for them.

Reply:
Our main priority should be the `scylla` crate. We don't give as much attention or care about stability of the `scylla-cql`'s API. The main reason we created it in the first place was because we needed it for `scylla-proxy`.

Reply:
I'll make the derive `cfg_attr(test)`, as it's needed for tests.